libfs/mount_interrupter.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libfs
+
+import (
+	"sync"
+)
+
+// MountInterrupter is for managing mounts with cancelation.
+type MountInterrupter struct {
```

Comment:
This type isn't really mounter specific, so I think a more generic name like `LazyOnceWaitable` would work as well. But I'm OK with the `MountInterrupter` too; just thought I'd point it out.